docker/build-images

```diff
@@ -4,8 +4,9 @@
 
 set -euo pipefail
 
-nix develop .#staticShell -c cargo build --profile=release-lto
+nix --extra-experimental-features nix-command --extra-experimental-features flakes develop .#staticShell -c cargo build --profile=release-lto 
```

Comment:
Thanks for the PR!

I think it can be written a bit more concisely with `--experimental-features "nix-command flakes"`, like we do in the `flake.nix`. Or do you still get a warning if you run this script with an activated the nix dev shell?

Reply:
Looks good, `--experimental-features "nix-command flakes"` is much cleaner.

I try to run `nix develop` and I got the same error:
```bash
$ nix develop 
error: experimental Nix feature 'nix-command' is disabled; use '--extra-experimental-features nix-command' to override
```

Reply:
That is expected if you haven't enable nix-command as documented here: https://nixos.wiki/wiki/Flakes

In that case you can activate the development shell with `nix-shell`. In that shell nix should then have the nix-command feature enabled.